publications-2025.bib

```diff
@@ -1665,6 +1708,22 @@ @Article{2025:seelinger.reinarz.ea:democratizing
   preprint = {https://arxiv.org/abs/2402.13768}
 }
 
+@Article{2025:shirin-t.reville.ea:spectral,
+  author  = {Shirin T, Asma and Reville, Brian and Schween, Nils W and Schulze, Florian and Kirk, John G},
+  title   = {Spectral curvature and breaks from Fermi acceleration at oblique shocks},
+  journal = {Monthly Notices of the Royal Astronomical Society: Letters},
+  year    = 2025,
+  volume  = 544,
+  number  = 1,
+  pages   = {L160--L166},
+  month   = nov,
+  issn    = {1745-3933},
+  url     = {https://academic.oup.com/mnrasl/article/544/1/L160/8339515},
+  doi     = {10.1093/mnrasl/slaf113},
+  publisher = {Oxford University Press (OUP)},
```

Comment:
```suggestion
  publisher = {Oxford University Press},
```

Is the OUP necessary?

Reply:
The publisher field will be removed as discussed on matrix.